Header header

bool fork
bool stopline
bool lost

int16 deviation
int16 oldDeviation